config/locales/views/advice_pages/heating_control.yml

```diff
@@ -57,6 +57,16 @@ en:
         view_our_analysis_html: <a href="%{href}">View our analysis</a>
         warm_weather:
           title: Leaving the heating on during warm weather
+      not_enough_data:
+        p1: When we have additional data for your school we will be able to provide you with detailed analysis which will
```

Comment:
Reviewing the text again, can we please include the `not_enough_data.available_from` message after these bullet points, so it described what will be in the analysis and then "Assuming we continue to regularly receive data we expect this analysis to be available after %{available_from}.", before going into the "in the meantime" part.